[2024-12-19T08:47:42.846+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T08:47:42.882+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2021-12-13T00:00:00+00:00 [queued]>
[2024-12-19T08:47:42.894+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2021-12-13T00:00:00+00:00 [queued]>
[2024-12-19T08:47:42.896+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T08:47:42.915+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2021-12-13 00:00:00+00:00
[2024-12-19T08:47:42.926+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1024) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T08:47:42.928+0000] {standard_task_runner.py:72} INFO - Started process 1036 to run task
[2024-12-19T08:47:42.931+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2021-12-13T00:00:00+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpi1ze4g8v']
[2024-12-19T08:47:42.935+0000] {standard_task_runner.py:105} INFO - Job 139: Subtask first_task
[2024-12-19T08:47:42.996+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2021-12-13T00:00:00+00:00 [running]> on host 8d142e9c83a7
[2024-12-19T08:47:43.092+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2021-12-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-13T00:00:00+00:00'
[2024-12-19T08:47:43.093+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T08:47:43.112+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T08:47:43.114+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T08:47:43.128+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T08:47:43.129+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T08:47:43.130+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T08:47:43.156+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T08:47:43.158+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2021-12-13T00:00:00+00:00, execution_date=20211213T000000, start_date=20241219T084742, end_date=20241219T084743
[2024-12-19T08:47:43.189+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T08:47:43.205+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T08:47:43.213+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
